src/6_branch.js

```diff
@@ -67,6 +67,10 @@ var wrap = function(parameters, func, init) {
 			 * @type {function(?Error,?): undefined}
 			 */
 			var done = function(err, data) {
+				if (err) {
```

Comment:
@Kirkkt Yup I tried that, what happens if there are no callbacks is that this thread stops execution with a `throw` so never gets to the `next()` call later on. Since there is one dead task at the head of the queue at that point, any other tasks added later would not be fired off.